geoapps/drivers/base_inversion.py

```diff
@@ -379,6 +381,9 @@ def get_tile_misfits(self, tiles):
             lsim, lmap = self.inversion_data.simulation(
                 self.mesh, self.active_cells, lsurvey, tile_id
             )
+
+            # TODO Parse workers to simulations
```

Comment:
Is this still a TODO?  aren't you assigning workers to the simulation in the next line?

Reply:
Yes, will need to assign a specific worker to simulations. Right now it's just None anyway